samples/notebooks/dotnet/09-BingSearch-using-kernel.ipynb

```diff
@@ -0,0 +1,224 @@
+    "using Microsoft.SemanticKernel.SkillDefinition;\n",
+    "using Microsoft.SemanticKernel.Orchestration;\n",
+    "using Microsoft.SemanticKernel.Planning;\n",
+    "using Microsoft.SemanticKernel.Planning.Sequential;\n",
+    "using Microsoft.SemanticKernel.TemplateEngine;\n",
+    "var builder = new KernelBuilder();\n",
+    "\n",
+    "// Configure AI backend used by the kernel\n",
+    "var (useAzureOpenAI, model, azureEndpoint, apiKey, orgId) = Settings.LoadFromFile();\n",
+    "\n",
+    "if (useAzureOpenAI)\n",
+    "    builder.WithAzureTextCompletionService(model, azureEndpoint, apiKey);\n",
+    "else\n",
+    "    builder.WithOpenAITextCompletionService(model, apiKey, orgId);\n",
+    "\n",
+    "var kernel = builder.Build();"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Add the following namespaces to use the Bing Search connector."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "dotnet_interactive": {
+     "language": "csharp"
+    },
+    "polyglot_notebook": {
+     "kernelName": "csharp"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "using Microsoft.SemanticKernel.Skills.Web;\n",
+    "using Microsoft.SemanticKernel.Skills.Web.Bing;"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Below are some examples on how `WebSearchEngineSkill` can be use in SK . Just uncomment the example to use the ."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {
+    "dotnet_interactive": {
+     "language": "csharp"
+    },
+    "polyglot_notebook": {
+     "kernelName": "csharp"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Example 1\n",
+      "What is quantum tunnelling\n",
+      "----\n",
+      "In physics, quantum tunnelling, barrier penetration, or simply tunnelling is a quantum mechanical phenomenon in which an object such as an electron or atom passes through a potential energy barrier that, according to classical mechanics, the object does not have sufficient energy to enter or surmount.\n",
+      "\n",
+      "Example 2\n",
+      "[1] Elon Musk (@elonmusk) is the owner of Twitter and is the most-followed person on Twitter, with over 144 million followers. Former U.S. President Barack Obama (@BarackObama) is the second most-followed person on Twitter, with over 132 million followers.\n"
+     ]
+    }
+   ],
+   "source": [
+    "// Load Bing skill\n",
+    "//Uncomment the Example \n",
+    "using (var bingConnector = new BingConnector(\"BING_KEY\")\n",
+    "{\n",
+    "kernel.ImportSkill(new WebSearchEngineSkill(bingConnector), \"bing\");\n",
+    "//First run the example sample given below and uncomment the await the  following statements \n",
+    "//await Example1Async(kernel);\n",
+    "//await Example2Async(kernel);\n",
```

Comment:
The cells where `Example1Async` and `Example2Async` are defined must be run before this cell in order for it to work, so please move those cells above this one.

Reply:
Also, another nit on spacing, but please indent inside the brackets.

Reply:
Reshuffled the cells as required